packages/pdf-mix-tool-git/pdf-mix-tool-git.pacscript

```diff
@@ -7,6 +7,7 @@ breaks="${_name} ${_name}-bin ${_name}-app ${_name}-deb"
 description="a GUI application to split, merge, rotate and mix PDF files"
 optdepends=("ImageMagick: image manipulation tool")
 maintainer="echometerain <echometer@disroot.org>"
+repology="pdfmixtool"
```

Comment:
Not eligible for `pacup`, so remove this.

Reply:
wdym?

Reply:
oh

Reply:
git package